sys/netpfil/pf/if_pfsync.c

```diff
@@ -1831,7 +2003,10 @@ pfsync_defer_tmo(void *arg)
 		free(pd, M_PFSYNC);
 	PFSYNC_BUCKET_UNLOCK(b);
 
-	ip_output(m, NULL, NULL, 0, NULL, NULL);
+	if (sc->sc_sync_peer.sa.sa_family == AF_INET)
```

Comment:
Please add `ifdef …` for address families. You might need to use a `switch` expression for that.

Reply:
Done